imports/startup/server/migrations.js

```diff
@@ -291,11 +291,11 @@ Migrations.add({
         { multi: true }
       );
     }
-    upgrade(Partners, 'relation');
     upgrade(Txdefs, 'data.relation');
     upgrade(Transactions, 'relation', { category: 'bill' });
     upgrade(Transactions, 'relation', { category: 'payment' });
     upgrade(Transactions, 'relation', { category: 'receipt' });
+    Partners.update({ relation: 'parcel' }, { $set: { relation: ['member'] } }, { multi: true });
```

Comment:
Miert erdemes beleirni egy korabbi migracioba? Ha mar tobbet sosem fut le, hisz adatbazisaink mind 23-as verzion vannak.
Egyebkent ha le is futna, akkor is egy 14-es allapotu db-n futna a 15-os migracios lepes, tehat ott meg nem elvaras hogy tomb legyen a relation. Es a 16-os lepes lehet hogy nem is ertene ugy a db strukturat, hiszen idoben csak kesobb tortenik majd, a 23-as lepesnel a tombosites.

Reply:
Ha esetleg valaki meteor resetet nyom, akkor bizony lefut. Mivel a mostani kódban a sémában a relationnek arraynek kell lennie, ezért nem tudjuk nem array-re updatelni. 16-22 verzióban partner kollekcióhoz nem nyúlunk, nem tudom, más hivatkozhat-e a partner.relationre.

Reply:
Hogy ez eddig nem realizaolodott bennunk. :)
A SimpleSchema hasznalata a db-be iraskor (tehat a Collection2 package hasznalata) nem osszeegyeztetheto a migracioval.
Ugyanis a 15-os migracionak ugy kene lefutnia hogy elotte a 14-es schemanak megfelelo objektumok vannak a db-ben, a migracio vegrehajtasa utan pedig a 15-os schema-nak megfelelo db-t kapunk.
De ha mindezt akkor akarjuk futtatni amikor kodunk mar a 23-as schema-nal jar, akkor nem tudjuk, hiszen kellene hozza a 14-es es a 15-os schemat ismerni. A schema-kat viszont historikusan nem taroljuk le verziozva.
Mit jelent ez szamunkra.
Hat azt hogy migracio alatt nincs ertelme schema-kat csekkolnunk hisz nem all rendelkezesunkre a megfelelo schema. Hanem minden db muveletet direct-tel kell meghivni kozvetlenul a kollekcion.

Reply:
Mondjuk reset utan elvileg uresnek kene lennie a db-nek, szova a semmit migraljuk, ami nem ertem hogy tud schemaba utkozni.
Lehet eleg az ilyen multi-s update-ejet hivni direct-tel?